src/main/java/net/minestom/server/instance/Chunk.java

```diff
@@ -400,25 +399,18 @@ public ChunkDataPacket getFreshPartialDataPacket() {
     }
 
     /**
-     * Gets the light packet of this chunk.
-     *
-     * @return the light packet
-     */
-    @NotNull
-    public UpdateLightPacket getLightPacket() {
-        ChunkLight lightData = new ChunkLight(this);
-        chunkLightEngine.lightChunk(lightData);
-        return lightData.convertDataIntoPacket();
-    }
-
-    /**
-     * Sets the chunk light engine
+     * Gets the {@link ChunkLight} at the given {@link Chunk}.
      * 
-     * @param chunkLightEngine The ChunkLightEngine implementation
+     * @param chunk the current chunk
+     * @return the chunk light data for a given chunk
      */
     @NotNull
-    public void setChunkLightEngine(@Nullable ChunkLightEngine chunkLightEngine) {
-        this.chunkLightEngine = chunkLightEngine;
+    public UpdateLightPacket getChunkLightPacket()
+    {
+        ChunkLightEngine engine = chunkLightEngine;
```

Comment:
forgot to remove this, as it's very unneeded. Was doing something different at the time, but forgot about this.